spec/features/aquariums/show_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe 'aquarium show ' do
+
+before(:each) do
+
+@shedd = Aquarium.create!(name: 'Shedd Aquarium', clean: true, number_of_types: 100)
+@national = Aquarium.create!(name: 'The National Aquarium', clean: true, number_of_types: 200)
+@denver = Aquarium.create!(name: 'Denver Aquarium', clean: true, number_of_types: 50)
+end
+
+  scenario 'visitor sees aquarium attributes on specific aquarium show page' do
+    visit "/aquariums/#{@shedd.id}"
+
+    expect(page).to have_content("#{@shedd.name}")
```

Comment:
Is there a reason these have to be entered as concatenated strings?

Reply:
Nope, I found unnecessary places elsewhere that I did that-- guess I just got string interpolation happy there.